wallet/src/blockchain.rs

```diff
@@ -1,14 +1,19 @@
+use chain_addr::Discrimination;
+use chain_impl_mockchain::vote::CommitteeId;
 use chain_impl_mockchain::{
     block::Block,
     fee::FeeAlgorithm as _,
     ledger::{Error, Ledger, LedgerParameters, LedgerStaticParameters},
     transaction::Input,
 };
```

Comment:
```suggestion
use chain_impl_mockchain::{
    block::Block,
    fee::FeeAlgorithm as _,
    ledger::{Error, Ledger, LedgerParameters, LedgerStaticParameters},
    transaction::Input,
    vote::CommitteeId,
};
```

Reply:
Auto import, I need to have more attention to this stuff.